jekyll/_cci2/building-docker-images.md

```diff
@@ -74,10 +75,18 @@ jobs:
 Below is a break down of what is happening during this build’s execution:
 
 * All commands are executed in the [primary-container](/docs/glossary/#primary-container). (line 5)
-* Once `setup_remote_docker` is called, all Docker-related commands are executed locally. (line 11)
-* [Docker Layer Caching](/docs/glossary/#docker-layer-caching) (DLC) is enabled to speed up image building. (line 13)
+* Once `setup_remote_docker` is called, all Docker-related commands are executed locally. (line 12)
+* [Docker Layer Caching](/docs/glossary/#docker-layer-caching) (DLC) is enabled to speed up image building. (line 14)
 * We use [project environment variables](/docs/set-environment-variable/#set-an-environment-variable-in-a-project) to store credentials for Docker Hub. (line 19)
 
+### Resource classes
+{: #resource-classes}
+
+**Arm on Docker** Support for Arm architecture in the Docker execution environment is in **Preview**. For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Arm and Docker Discuss post](https://discuss.circleci.com/t/product-launch-arm-docker-preview/48601).
+{: class="alert alert-caution"}
+
+The resource class specified for the primary container also applies to the remote Docker environment. Both x86 and Arm resource classes are available. For a full list of available resource classes see the [Configuration Reference](/docs/configuration-reference/#docker-execution-environment).
```

Comment:
i thought about this some more and realized that this is not actually accurate.  for x86, if you specify a docker.xlarge & setup_remote_docker, the primary container & the VM are going to be 8 CPU / 32 GB RAM, not 8 CPU 16 GB RAM. https://circleci.com/product/features/resource-classes/

for ARM, we will use the ARM VM equivalent size when you use setup_remote_docker

Reply:
Ahhh ok I see, I'll take another look at this section